project/cmd.go

```diff
@@ -6,25 +6,37 @@ import (
 )
 
 var Flags struct {
-	Path  string
-	Debug bool
-	Tag   string
+	Path          string
+	Debug         bool
+	Tag           string
+	SecretsInput  string
+	SecretsOutput string
+	FailureOutput string
```

Comment:
why do we need both SecretOutput and FailureOutput? do we have a use case need to write them to a different place?